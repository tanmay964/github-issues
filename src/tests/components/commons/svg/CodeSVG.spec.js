import React from 'react';
import renderer from 'react-test-renderer';
import CodeSVG from '../../../../components/commons/svg/CodeSVG';

it('renders correctly', () => {
  const tree = renderer
    .create(<CodeSVG />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
