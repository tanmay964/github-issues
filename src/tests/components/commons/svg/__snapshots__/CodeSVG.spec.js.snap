// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<svg
  aria-hidden="true"
  className="sc-bdVaJa ijurAl"
  height="16"
  width="14"
>
  <path
    d="M9.5 3L8 4.5 11.5 8 8 11.5 9.5 13 14 8 9.5 3zm-5 0L0 8l4.5 5L6 11.5 2.5 8 6 4.5 4.5 3z"
    fillRule="evenodd"
  />
</svg>
`;
